MAUI/DateTimePicker/customizations.md

```diff
@@ -0,0 +1,337 @@
+The SfDateTimePicker control allows you to customize the footer divider color by setting the [DividerColor] property of [PickerFooterView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfDateTimePicker x:Name="picker" >
+    <picker:SfDateTimePicker.FooterView >
+        <picker:PickerFooterView DividerColor="#A9A9A9" />
+    </picker:SfDateTimePicker.FooterView>
+</picker:SfDateTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" %}
+
+SfDateTimePicker picker = new SfDateTimePicker();
+picker.FooterView = new PickerFooterView()
+{
+    DividerColor = Color.FromArgb("#A9A9A9")
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customization of footer
+You can customize the footer text style and background color of the `Date Time Picker` using the [TextStyle] and [Background] properties of the [PickerFooterView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfDateTimePicker x:Name="picker" >
+    <picker:SfDateTimePicker.FooterView >
+        <picker:PickerFooterView Background="#6750A4">
+            <picker:PickerFooterView.TextStyle >
+                <picker:PickerTextStyle FontSize="15" TextColor="White" />
+            </picker:PickerFooterView.TextStyle>
+        </picker:PickerFooterView>
+    </picker:SfDateTimePicker.FooterView>
+</picker:SfDateTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" %}
+
+SfDateTimePicker picker = new SfDateTimePicker();
+picker.FooterView = new PickerFooterView()
+{
+    Background = Color.FromArgb("#6750A4"),
+    TextStyle = new PickerTextStyle()
+    {
+        TextColor = Colors.White,
+        FontSize = 15,
+    }
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Selection View Customization
+You can customize the Date Time Picker selection view by using the `SelectionView` property of `SfDateTimePicker`.
+
+### Set selection view
+In the SfDateTimePicker control, corner radius, stroke , and padding can be customized by setting the [CornerRadius], [Stroke] and [Padding] properties in the [PickerSelectionView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfDateTimePicker x:Name="picker" >
+    <picker:SfDateTimePicker.SelectionView >
+        <picker:PickerSelectionView CornerRadius="5" Stroke="Violet" Padding="5, 10, 5, 10" Background="#6750A4" />
+    </picker:SfDateTimePicker.SelectionView>
+</picker:SfDateTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" %}
+
+SfDateTimePicker picker = new SfDateTimePicker();
+picker.SelectionView = new PickerSelectionView()
+{
+    CornerRadius = 5,
+    Stroke = Colors.Violet,
+    Pading = new Thickness(5, 10, 5, 10),
+    Background = Color.FromArgb("#6750A4"),
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customization of selected item
+You can customize the selected view text style of the `Date Time Picker` using the [SelectedTextStyle] property of the [SfDateTimePicker].
```

Comment:
Customize the selected view text style of the `Date Time Picker` using the [SelectedTextStyle] property of the [SfDateTimePicker].